components/fieldgroups/experience-event/experienceevent-campaign-delivery-log.schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-campaign-delivery-log",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Campaign Marketing Delivery Logs",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Campaign marketing events such as opens, clicks, sent, opt-out, links and impressions.",
+  "definitions": {
+    "experienceevent-campaignlog": {
+      "properties": {
+        "xdm:externalId": {
+          "title": "External Identifier",
+          "type": "string",
+          "description": "External Identifier that defines the primary identity value for profile and audience."
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Identifier of the marketing campaign to which activity originating this message belongs to."
+        },
+        "xdm:campaignName": {
+          "title": "Campaign Name",
+          "type": "string",
+          "description": "Internal Name that uniquely identifies the Campaign."
+        },
+        "xdm:campaignLabel": {
+          "title": "Campaign Label",
+          "type": "string",
+          "description": "A human-friendly identifier of the marketing campaign which is originating this message."
+        },
+        "xdm:broadlogId": {
+          "title": "Broad Log Identifier",
+          "type": "string",
+          "description": "Identifier of the marketing campaign delivery log this event belongs to."
+        },
+        "xdm:deliveryId": {
+          "title": "Delivery Identifier",
+          "type": "string",
+          "description": "The campaign activity originating this message."
+        },
+        "xdm:visitorId": {
+          "title": "Visitor Identifier",
+          "type": "string",
+          "description": "Visitor that was targeted as part of the delivery."
+        },
+        "xdm:recipientId": {
+          "title": "Recipient Identifer",
+          "type": "string",
+          "description": "Recipient that was targeted as part of this delivery."
+        },
+        "xdm:serviceId": {
+          "title": "Service Identifier",
+          "type": "string",
+          "description": "Subscribed Service that targeted this delivery."
+        },
+        "xdm:channel": {
+          "title": "Channel",
+          "type": "string",
+          "description": "Channel of the delivery.",
+          "meta:enum": {
```

Comment:
Add an actual "enum" attribute if you want this field constrained to specific values only.

Reply:
Same reason as above